### DISCUSSION TITLE
Is there any current mechanism for generating a manifest of the built output?

### DISCUSSION THREAD

**Discussion by charrondev** [vercel/turborepo#3742] [Help] [answered]
One of the very useful things that we get from webpack is the ability to dump a JSON manifest that dumps the chunk graph. In particular, when there are async chunks, it outputs which JSON chunks depend on other chunks.

This is particularly useful generating preload headers for a page. I'm assuming NextJS relies on a similar type of mechanism so I was wondering if there was anything exposed in this respect that could useful.

**Comment by anthonyshew** [comment, marked as answer]
You'll find chunking information (manifests, chunks, etc) in `.next` after you've done a build, yes.
